src/Tools/Source/RunTests/Program.cs

```diff
@@ -42,7 +44,9 @@ internal static int Main(string[] args)
                 cts.Cancel();
             };
 
-            return Run(options, cts.Token).GetAwaiter().GetResult();
+            var result = Run(options, cts.Token).GetAwaiter().GetResult();
+            CheckTotalDumpFilesSize();
```

Comment:
@jaredpar In this case is @ivanbasov adding the third case: dumping for the VS process as well. The cases that we've screwed up in the past is say we're crashing on VS startup, so we create a dump, that test fails, we launch VS again, create a dump, rinse, repeat. We've had cases where I think we're uploading 10s of gigabytes of dump files _per build_.

We don't intend this limit to apply to reasonable cases, only accidentally unreasonable ones. @mmitche can explain more about storage consumption we've done in the past.